db/migrate/20230523134506_claim_details.rb

```diff
@@ -4,8 +4,9 @@ def change
     add_column :claims, :defence_statement, :string
     add_column :claims, :number_of_witnesses, :integer
     add_column :claims, :supplemental_claim, :string
-    add_column :claims, :preparation_time, :string
     add_column :claims, :time_spent_hours, :integer
     add_column :claims, :time_spent_mins, :integer
+    add_column :claims, :work_before_date, :date
+    add_column :claims, :work_after_date, :date
```

Comment:
This is will break the server as this migration has already been run in the past, so the new fields WILL NOT be added to the database.

The the following in the console:
```
rails g migration <name of migration>
```

This will generate a new file in then `db` directory and so can add the column changes there...